companion/src/cmake/Macros.cmake

```diff
@@ -24,3 +24,15 @@ MACRO (NOW RESULT)
     ENDIF (WIN32)
 ENDMACRO (NOW)
 
+MACRO (GIT_ID RESULT)
+    IF (WIN32)
+        EXECUTE_PROCESS(COMMAND "cmd" " /C git rev-parse HEAD" OUTPUT_VARIABLE ${RESULT})
```

Comment:
Just a cosmetics comment here, I prefer lowercase for commands such as MACRO or IF or EXECUTE_PROCESS (but not COMMAND)